womtool/src/main/scala/cromwell/pipeline/womtool/WomTool.scala

```diff
@@ -11,12 +11,7 @@ import spray.json.DefaultJsonProtocol._
 import spray.json._
 import wom.executable.WomBundle
 import wom.expression.WomExpression
-import wom.graph.{
-  ExternalGraphInputNode,
-  OptionalGraphInputNode,
-  OptionalGraphInputNodeWithDefault,
-  RequiredGraphInputNode
-}
+import wom.graph.{ ExternalGraphInputNode, OptionalGraphInputNode, OptionalGraphInputNodeWithDefault, RequiredGraphInputNode }
```

Comment:
Observing this case, I suppose, that 150 limits would be better than 160, I don't know, I think we should discuss it

Reply:
Agree with Ostap.

Reply:
The same as previous :)